### PR TITLE
Changes on readme

### LINKED ISSUE #2: typo errors in README.md
[configuring-pagination-items-per-page](https://github.com/UlisesGascon/express-simple-pagination#configuring-pagination-items-per-page) example doesn't pass options to middleware function.

Also all `server.listen` callbacks log that the listening port is 80 although 3000 has been established.

---
En el ejemplo `configuring-pagination-items-per-page` no se pasa las opciones al middleware.

Además, en todos los ejemplos, los callbacks de `server.listen` siempre se imprime que se va a escuchar el puerto 80, cuando según el ejemplo el puerto usado es el 3000.

### PR DESCRIPTION
**Pull request by devgar**
Fixes some typoe errors. Fix #2

The important change and motivation of this pull request is in line 135

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -132,12 +132,12 @@ const paginationOptions = {
   max: 1000
 }
 
-app.get('/products', pagination(), (req, res, next) => {
+app.get('/products', pagination(paginationOptions), (req, res, next) => {
```
Comment (UlisesGascon) [inline README.md:138]: Wow! Great Catch! 👍

File: README.md [hunk 2]
```diff
@@ -68,8 +68,8 @@ app.get('/products', (req, res, next) => {
   res.json({msg: req.pagination})
 })
 
-app.listen(3000, () => {
-  console.log('web server listening on port 80')
+const server = app.listen(3000, () => {
```
Comment (UlisesGascon) [inline README.md]: Great catch! Thanks a lot for this PR! @devgar 

Maybe can we use just `port`? I love the idea of using `server.address().port` but not all the developers are used to it. 

```js
const port = 3000;

app.listen(port, () => {
  console.log(`web server listening on port ${port}`)
})
```

What do you think?

**Review by UlisesGascon — CHANGES_REQUESTED** [review]

**Comment by devgar** [comment]
Ok, It is better.
My proposal was needlessly too complex to use it in examples.

**Review by UlisesGascon — APPROVED** [review]

**Comment by UlisesGascon** [comment]
This will be release v1.0.2. thanks a lot @devgar !

**Pull request merged**